packages/core/src/data-grid-overlay-editor/data-grid-overlay-editor.tsx

```diff
@@ -71,13 +71,21 @@ const DataGridOverlayEditor: React.FunctionComponent<DataGridOverlayEditorProps>
     let pad = true;
     let editor: React.ReactNode;
     if (CustomEditor !== undefined) {
+        const onCustomFinishedEditing = React.useCallback((newValue?: any | undefined) => {
+            if (newValue !== undefined) {
+                onFinishEditing(newValue, [0, 0]);
+                return;
+            }
+
+            onClickOutside();
+        }, [onFinishEditing]);
```

Comment:
- `onClickOutside` just calls `onFinishEditing` with `tempValue`, so maybe this could be consolidated into `onFinishEditing(newValue !== undefined ? newValue : tempValue, [0, 0])`?
- `onClickOutside` should be a dependency as well

Reply:
Good point. Done!